config/cloud.cfg.tmpl

```diff
@@ -38,6 +48,24 @@ preserve_hostname: false
 # This should not be required, but leave it in place until the real cause of
 # not finding -any- datasources is resolved.
 datasource_list: ['NoCloud', 'ConfigDrive', 'Azure', 'OpenStack', 'Ec2']
+{% elif variant in ["photon"] %}
```

Comment:
the datasource_list above was added to work around a specific bug. Unless you have datasources not being detected, I think you can get rid of this section.

Reply:
We need this, `VMwareGuestInfo` this is not available by default and we want to go in the order given in the list.

If you insist, I will remove this.

Reply:
It is ok to keep (though I'd remove the comments). It'd be nice to get to the point where we don't need it though.

Reply:
I understand. The reason why I have kept the commented items is, we run some tests on azure, gce, aws and our test scripts enables desired entry. I need to fix things in multiple places, it's okay. I will remove the commented ones. Thanks.